src/Data/Models/ChannelOperationRequest.cs

```diff
@@ -57,7 +57,12 @@ public enum ChannelOperationRequestStatus
         /// <summary>
         /// The process failed in background
         /// </summary>
-        Failed = 8
+        Failed = 8,
+
+        /// <summary>
+        /// The PSBT is being funded and signed by NodeGuard
+        /// </summary>
+        FinalizingPSBT = 9
```

Comment:
```suggestion
        /// <summary>
        /// The PSBT is being signed by NodeGuard after all human required signatures have been collected
        /// </summary>
        FinalizingPSBT = 9
```

So this is basically, all human cosigners did they work, now its time for NG to sign the funded PSBT which has been validated by LND.